locations/spiders/dillards.py

```diff
@@ -4,46 +4,41 @@
 from scrapy.spiders import SitemapSpider
 
 from locations.hours import OpeningHours
-from locations.items import Feature
+from locations.structured_data_spider import StructuredDataSpider
 
 
-class DillardsSpider(SitemapSpider):
+class DillardsSpider(SitemapSpider, StructuredDataSpider):
     name = "dillards"
     item_attributes = {"brand": "Dillard's", "brand_wikidata": "Q844805"}
     allowed_domains = ["www.dillards.com"]
     sitemap_urls = ("https://www.dillards.com/sitemap/sitemap_storeLocator_1.xml",)
```

Comment:
While we are here may as well update this `https://www.dillards.com/robots.txt` or `https://www.dillards.com/sitemap/sitemap.xml` in case we get `_2.xml`. `sitemap_filter` can restrict which ones we download, `sitemap_rules` may also want to be updated it seems the urls now include 3 `/`s.